pkg/sentry/fs/proc/sys_net.go

```diff
@@ -280,11 +280,120 @@ func (p *proc) newSysNetCore(ctx context.Context, msrc *fs.MountSource, s inet.S
 	return newProcInode(ctx, d, msrc, fs.SpecialDirectory, nil)
 }
 
+// ipForwarding implements fs.InodeOperations.
+// ipForwarding is used to enable/disable packet forwarding of netstack
+// and cache user input to "/proc/sys/net/ipv4/ip_forward".
+//
+// +stateify savable
+type ipForwarding struct {
+	stack inet.Stack `state:"wait"`
+	fsutil.SimpleFileInode
+
+	// mu protects enabled
+	mu sync.Mutex
+
+	// Linux stores these as an integer,
+	// so if you write "2" into ip_forward, you should get "2" back.
+	// For compatibility reason enabled is stored as an int32
```

Comment:
Missing period at the end of sentence. Same for the others.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
fixed